c7n/resources/redshift.py

```diff
@@ -245,6 +245,38 @@ def process_cluster_snapshot(self, cluster):
             ClusterIdentifier=cluster['ClusterIdentifier'])
 
 
+@actions.register('enhanced-vpc-routing')
```

Comment:
one minor preference for actions to be verbs, where as this is a noun (delta to goal state based systems versus imperative)... so enable-vpc-routing with the boolean as 'value' versus enabled since we're tending towards the later as a common vocabulary for switches that go true/false. albeit that's not a great nomenclature either.